ethexe/rpc/src/apis/dev.rs

```diff
@@ -0,0 +1,57 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2024 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use std::sync::Arc;
+
+use ethexe_observer::MockBlobReader;
+use gear_core::ids::prelude::CodeIdExt;
+use gprimitives::{CodeId, H256};
+use jsonrpsee::{
+    core::{async_trait, RpcResult},
+    proc_macros::rpc,
+};
+use sp_core::Bytes;
+
+#[rpc(server)]
+pub trait Dev {
+    #[method(name = "blob_set")]
+    async fn blob_set(&self, blob: Bytes) -> RpcResult<(H256, CodeId)>;
```

Comment:
Otherwise function call looks like it's already happened 
```suggestion
    async fn set_blob(&self, blob: Bytes) -> RpcResult<(H256, CodeId)>;
```